lib/Statocles/Plugin/HTMLLint/RexOps.pm

```diff
@@ -0,0 +1,87 @@
+package Statocles::Plugin::HTMLLint::RexOps;
+our $VERSION = '0.001';
+# ABSTRACT: Check HTML for common errors and issues for the rexify-website
+
+use Statocles::Base 'Class';
+with 'Statocles::Plugin';
+BEGIN {
+    eval { require HTML::Lint::Pluggable; HTML::Lint::Pluggable->VERSION( 0.06 ); 1 }
+        or die "Error loading Statocles::Plugin::HTMLLint. To use this plugin, install HTML::Lint::Pluggable";
+};
+
+=attr plugins
+
+The L<HTML::Lint::Pluggable> plugins to use. Defaults to a generic set of
+plugins good for HTML5: 'HTML5' and 'TinyEntitesEscapeRule'
```

Comment:
`TinyEntitesEscapeRule` instead of `TinyEntitiesEscapeRule` ?

Reply:
Yep, noticed that too, but seems like valid: https://metacpan.org/pod/release/KARUPA/HTML-Lint-Pluggable-0.10/lib/HTML/Lint/Pluggable/TinyEntitesEscapeRule.pm

Reply:
ouch. Agreed.